core/templates/pages/story-editor-page/story-editor-page.component.spec.ts

```diff
@@ -114,16 +114,16 @@ describe('Story editor page', function() {
       thumbnail_filename: null,
       url_fragment: 'story-url-fragment'
     });
-    var MockEditableStoryBackendApiService = {
-      validateExplorations: () => Promise.resolve([])
+    const MockEditableStoryBackendApiServiceAsync = {
```

Comment:
This is not how its done... The service is not async, it is the function. So the function name needs async, and it needs to be updated in all places where the function is called.

Reply:
Thank you very much for the guidance, I re read the problem statement and I think now it's good.